sklearn/utils/estimator_checks.py

```diff
@@ -2279,6 +2274,16 @@ def enforce_estimator_tags_y(estimator, y):
     return y
 
 
+def enforce_estimator_tags_X(estimator, X):
+    # Estimators with a `requires_positive_X` tag only accept strictly positive
+    # inputs
+    if _safe_tags(estimator, "requires_positive_X"):
+        # Create strictly positive X. The minimal increment above 0 is 1, as
```

Comment:
The comment and the code don't really agree, right?

Reply:
That's right, I forgot the comment :p (see https://github.com/scikit-learn/scikit-learn/pull/14705#discussion_r326872132)